src/cuda_eval.cpp

```diff
@@ -214,7 +217,7 @@ void jitc_cuda_assemble(ThreadState *ts, ScheduledGroup group,
                 params_type, params_base, v, v);
 
             if (vt != VarType::Bool) {
-                fmt("    st.global.cs.$t [%rd0], $v;\n", v, v);
+                fmt("    st.global.cs.$T [%rd0], $v;\n", v, v);
```

Comment:
ditto

Reply:
I was trying to be a bit too clever by adding a special type character just to handle the fact PTX instructions for a lot of the data move/load/store operations can't be specified by a `fp16`  type and instead need `b16` (keeping all the other types the same). But I agree, just having `$b` for everything doesn't really hurt readability and reduces bloat so I've made that switch.